src/bosh_azure_cpi/lib/cloud/azure/disk/config_disk_manager.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require 'etc'
+
+module Bosh::AzureCloud
+  class ConfigDiskManager
+    include Helpers
+
+    BOSH_CONFIG_DISK_LABEL = 'azure_cfg_dsk'
+    CONFIG_DISK_CONTAINER  = 'azure-config-disks'
+    CONFIG_DISK_MOUNT_POINT = 'azure_config_disk_mount'
+    CONFIG_DISK_FILE_PATH_PREFIX = 'azure_config_disk_image'
+    CONFIGS_RELATIVE_PATH = 'configs'
+    CONFIG_DISK_SIZE = 24 # in MB, azure have a requirement for the min size.
+    MEGA_SIZE = 1024 * 1024
+    def initialize(blob_manager, disk_manager2, storage_account_manager)
+      @blob_manager = blob_manager
+      @disk_manager2 = disk_manager2
+      @storage_account_manager = storage_account_manager
+      @logger = Bosh::Clouds::Config.logger
+    end
+
+    # Returns the config disk uri.
+    def prepare_config_disk(resource_group_name, vm_name, location, metadata_obj, user_data_obj)
+      disk = create_config_disk(resource_group_name, vm_name, location, metadata_obj, user_data_obj)
+      disk
+    end
+
+    private
+
+    def create_config_disk(resource_group_name, vm_name, location, meta_data_obj, user_data_obj)
```

Comment:
to follow the same style, this should be with prefix _

Reply:
sure.